### PR TITLE
Add DatePicker for Start and End Period

### PR DESCRIPTION
**Pull request by steveoni**
This pr adds a new schema field (`start_period` and `end_period`) to dataset . The pr implements issue https://github.com/FCSCOpendata/ckanext-fcscopendata/issues/38 and also supports this pr https://github.com/FCSCOpendata/ckanext-fcscopendata/pull/44

### CODE REVIEW

File: ckanext/scheming/assets/webassets.yml [hunk 1]
```diff
@@ -21,6 +21,17 @@ multiple_text:
...
 
+mainschema-js:
+  filters: rjsmin
+  output: ckanext-scheming/%(version)s_main.js
+  extra:
+    preload:
+      - base/main
+  contents:
+    - vendor/jquery.js
+    - vendor/datepicker/js/bootstrap-datepicker.js
```
Comment (sagargg) [inline ckanext/scheming/assets/webassets.yml]: I don't see the `bootstrap-datepicker.js` in a  v2.9.4`.  Maybe you are using a different CKAN version. 

https://github.com/ckan/ckan/tree/ckan-2.9.4/ckan/public/base/vendor

**Review by sagargg — COMMENTED** [review]
@steveoni  For me, it is failing to locate the `bootstrap-datepicker.js` resource.  Could you check that?

File: ckanext/scheming/assets/webassets.yml (hunk 1 above)
Comment (steveoni) [reply ckanext/scheming/assets/webassets.yml]: i'm using ckan 2.9.4 can you verify yours

```json
{
  "help": "http://ckan-dev:3000/api/3/action/help_show?name=status_show",
  "success": true,
  "result": {
    "site_title": "CKAN",
    "site_description": "",
    "site_url": "http://ckan-dev:3000",
    "ckan_version": "2.9.4",
    "error_emails_to": null,
    "locale_default": "en",
    "extensions": [
      "envvars",
      "odata",
      "vocabulary",
      "image_view",
      "text_view",
      "datastore",
      "datapusher",
      "dataexplorer_view",
      "hierarchy_display",
      "dataexplorer_table_view",
      "dataexplorer_chart_view",
      "dataexplorer_map_view",
      "fcscopendata",
      "pages",
      "scheming_datasets",
      "scheming_groups",
      "scheming_organizations",
      "fluent",
      "cloudstorage",
      "asset_storage",
      "noanonaccess"
    ]
  }
}
```

Comment (sagargg) [reply ckanext/scheming/assets/webassets.yml]: **Fixed**

**Comment by sagargg** [comment]
**LGTM** 👍

**Pull request merged**